packages/kit/src/views/SignatureConfirm/pages/TxConfirm/TxConfirm.tsx

```
@@ -0,0 +1,256 @@
+  ]);
+
+  const txConfirmTitle = useMemo(() => {
+    if ((!decodedTxs || decodedTxs.length === 0) && !decodedTxsInit.current) {
+      return '';
+    }
+
+    if (
+      decodedTxs &&
+      decodedTxs[0] &&
+      decodedTxs[0].txDisplay &&
+      decodedTxs[0].txDisplay.title
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)